lib/dal/src/audit_logging.rs

```diff
@@ -184,50 +198,93 @@ pub async fn list(ctx: &DalContext) -> Result<Vec<FrontendAuditLog>> {
         })
         .await?;
 
-    // TODO(nick): remove hard-coded max messages value.
-    let mut messages = consumer.fetch().max_messages(200).messages().await?;
+    // TODO(nick): replace hard-coded max messages value with proper pagination.
+    let mut messages = consumer.fetch().max_messages(1000).messages().await?;
```

Comment:
Just a reminder that we will need to stop doing the "post-process" filter and pagination we are doing now and will need to replace it with a real solution. We use a non-durable, regular pull consumer now and we may need to use an ordered push consumer (also non-durable by its nature) when we do this.